web_01.html

```diff
@@ -0,0 +1,219 @@
+                <p>creative</p>
+              </div>
+            </div>
+            <div class="projects__content">
+              <a href="#" class="projects__image">
+                <img src="/images/antic_popcorn.jpg"/>   
+              </a>
+              <div class="projects__texts">
+                <span class="projects__left-border"></span>
+                <span class="projects__right-border"></span>
+                <h1>Antic Popcorn</h1>
+                <p>illustration</p>
+              </div>
+            </div>
+            <div class="projects__content">
+              <a href="#" class="projects__image">
+                <img src="/images/website_fonts.jpg"/>   
+              </a>
+              <div class="projects__texts">
+                <span class="projects__left-border"></span>
+                <span class="projects__right-border"></span>
+                <h1>Website Fonts</h1>
+                <p>typography</p>
+              </div>
+            </div>
+            <div class="projects__content">
+              <a href="#" class="projects__image">
+                <img src="/images/vectorized_ideas.jpg"/>   
+              </a>
+              <div class="projects__texts">
+                <span class="projects__left-border"></span>
+                <span class="projects__right-border"></span>
+                <h1>Vectorized Ideas</h1>
+                <p>illustration</p>
+              </div>
+            </div>
+            <div class="projects__content">
+              <a href="#" class="projects__image">
+                <img src="/images/logo_mockup.jpg"/>   
+              </a>
+              <div class="projects__texts">
+                <span class="projects__left-border"></span>
+                <span class="projects__right-border"></span>
+                <h1>Logo Mockup</h1>
+                <p>creative</p>
+              </div>
+            </div>
+          </div>
+          <a href="#" class="projects__button">SHOW ME MORE</a> 
+        </div>
+      </section>
+      <section class="parallax__background">
+        <p>Proudly serving over 1600 satisfied users!</p> 
+      </section>
+    </main>
+    <footer>
+      <div class="footer__top">
+        <div class="container">
+          <div>
+            <h2>About Us</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cumque unde, omnis, reiciendis, quisquam ea ab distinctio?</p>
+            <a href="#" id="footer__au-button">work with us</a>
+          </div>
+          <div>
+            <h2>Latest Tweet</h2>
+            <p>I've just taken the WordPress 2013 User and Developer Survey, have you?<br><a href="#">http://wp-survey.polldaddy.com/s/wp-2013</a> (pass it on!)</p>
+          </div>
+          <div>
+            <h2>Contact Info</h2>
+            <p><span>Phone:</span>+37 830 851 108</p>
+            <p><span>Address:</span>15 Mini Street, Chicago, USA</p>
+            <p><span>Fax:</span>+37 830 851 108</p>
```

Comment:
You can use `telno` in `href` which opens a default application for calls with pre-defined contact number.
```suggestion
            <p><span>Fax:</span><a href="tel:+37830851108">+37 830 851 108</a></p>
```